src/member.ts

```diff
@@ -88,20 +93,33 @@ export class Member implements IStateful<IMemberState> {
         ${Member.fragments.ReputationHolderFields}
       `
 
+      const itemMap = (item: any) => Member.itemMap(context, item)
+
       return context.getObservableList(
           query,
-          (r: any) => {
-            return new Member(context, {
-              address: r.address,
-              contract: r.contract,
-              reputation: new BN(r.balance)
-            })
-          },
+          itemMap,
```

Comment:
Let's modify getObservableList to take context as a paremter. This way we can just pass in Member.itemMap and we don't have to wrap it.

Reply:
Yeah I did this refactor, but the itemMap passed down has custom logic sometimes. Those ones (just a few) still needed a wrapper